### PR TITLE
bumped versions and updated code to pass linting

### PR DESCRIPTION
**Pull request by danielmoll**
Updated to react 0.14.0
Updated devpack to latest.
Updated code to pass linting - had to rename to `export default class FourOhFourPage ...` - updated css class as well to reflect name change to keep consistent.

### CODE REVIEW

**Comment by sebinsua** [comment]
This looks good to me.

**Comment by sebinsua** [comment]
@keithamus Can you look and merge?

File: example.es6 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import FourOFourPage from './index.es6';
+import FourOhFourPage from './index.es6';
 export default (
-  <FourOFourPage />
+  <FourOhFourPage />
```
Comment (keithamus) [inline example.es6]: If we're changing this can we just change it to `NotFoundHandler` please.

File: index.es6 [hunk 2]
```diff
@@ -1,34 +1,53 @@
...
+                68.5,324.9 C557.5,686,702.5,685.4,779.3,
+                597.7c69.1-79,67.1-204.6-70.4-324.8C557.8,
+                140.8,412.1,130.3,331.1,222.9`}
+              />
+
+              <path
+                fill="#4D4D4D"
+                d={`M1186.7,717.1h-129.5v169.3L932,915.5V717.1H520.2V624L932,
+                2.6h126.6V608h128.1V717.1z M932,608V152.5 L629.3,608H932z`}
+              />
```
Comment (keithamus) [inline index.es6]: Good. This is good.

**Comment by sebinsua** [comment]
@keithamus Can we merge now?

**Comment by keithamus** [comment]
Yup I'm very happy with this. But for some reason don't have access to merge.

**Pull request merged**